src/frontend/package.json

```diff
@@ -11,12 +11,13 @@
     "@types/react-modal": "^3.12.0",
     "axios": "^0.21.1",
     "class-transformer": "0.3.1",
-    "next": "10.0.6",
-    "react": "17.0.1",
+    "next": "^10.0.6",
+    "react": "^17.0.1",
     "react-circular-progressbar": "^2.0.3",
-    "react-dom": "17.0.1",
+    "react-dom": "^17.0.1",
     "react-input-slider": "^6.0.0",
-    "react-modal": "^3.12.1"
+    "react-modal": "^3.12.1",
+    "sass": "^1.32.8"
```

Comment:
What do these changes help with? I'm not opposed just wondering

Reply:
@zefwang lol what are these doing

Reply:
oh wait, i see you added sass and npm automatically updated the other versions. What's sass for again?